distribution.md

```diff
@@ -228,3 +230,89 @@ The Run Image SHOULD contain the following Labels on the image configuration:
 Where,
 
 `<Target ID>` is an identifier specified on the runtime image that MAY be used to apply target-specific logic.
+
+### Builder
+
+The following specifies the artifact format for a buildpacks builder.
```

Comment:
nit: sometimes we use "builder" and sometimes "Builder". We should probably pick one and run with it. I vote for "builder".

Reply:
Sure, I will make the changes